python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -481,12 +481,11 @@ def to_bundle(resource: ExecutionResources) -> Dict:
         return req
 
     for op, state in topology.items():
-        per_task_resource = op.incremental_resource_usage()
-        task_bundle = to_bundle(per_task_resource)
-        resource_request.extend([task_bundle] * op.num_active_work_refs())
```

Comment:
@ericl This looks like a bug. Could you help confirm? We multiple the per-task resource by `op.num_active_work_refs`, which is the number of submitted tasks. I don't think it makes any sense.

Reply:
I think this is needed, since the request_resources() API takes into account existing running tasks. I.e., if there are currently 5 CPUs used, and you want it to scale to 6 CPUs, you need to request 6 CPUs and not 1 CPU.

Reply:
oh, I see. Another issue is that `to_bundle` will round up the resources, and then multiply the resources by op.num_active_work_refs. I think we need to do multiplication before the round up.

Reply:
Just found that `AutoscalingRequester` will do round-up. So we can remove the round up in `to_bundle`?

Reply:
Then problem is that we setting per-task num_cpus to 0.1. It will still sum up too large.